<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallerManagerWidgetClass</class>
 <widget class="QWidget" name="CallerManagerWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CallerManagerWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_main">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lb_callerManager">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Caller Manager</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Search</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QGridLayout" name="gridLayout_2" columnstretch="10,5">
                <item row="1" column="0">
                 <widget class="QLineEdit" name="le_searchName">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                   </font>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0" colspan="2">
                 <widget class="QLabel" name="lb_costUnit">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                   </font>
                  </property>
                  <property name="text">
                   <string>CostUnit</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0" colspan="2">
                 <widget class="QLabel" name="lb_companyLocation">
                  <property name="text">
                   <string>Company Location</string>
                  </property>
                 </widget>
                </item>
                <item row="8" column="0" colspan="2">
                 <widget class="QComboBox" name="cb_companyLocation"/>
                </item>
                <item row="6" column="0" colspan="2">
                 <widget class="QComboBox" name="cb_searchCostUnit">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                   </font>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="lb_phone">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                   </font>
                  </property>
                  <property name="text">
                   <string>Phone</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="le_searchPhone">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                   </font>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="lb_name">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                   </font>
                  </property>
                  <property name="text">
                   <string>Name</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0" colspan="2">
                 <widget class="QLineEdit" name="le_searchDepartmnet">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                   </font>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0" colspan="2">
                 <widget class="QLabel" name="lb_department">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                   </font>
                  </property>
                  <property name="text">
                   <string>Department</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="chb_searchDeletedCaller">
                <property name="font">
                 <font>
                  <family>Arial</family>
                 </font>
                </property>
                <property name="text">
                 <string>Include Deleted Caller</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pb_reset">
                <property name="font">
                 <font>
                  <family>Arial</family>
                 </font>
                </property>
                <property name="text">
                 <string>Reset</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Actions</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="1" column="1">
                 <widget class="QPushButton" name="pb_editCaller">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                   </font>
                  </property>
                  <property name="text">
                   <string>edit</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="pb_deleteCaller">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                   </font>
                  </property>
                  <property name="text">
                   <string>delete</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0" colspan="2">
                 <widget class="QPushButton" name="pb_addNewCaller">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                   </font>
                  </property>
                  <property name="text">
                   <string>Add new</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="tv_callerTable">
           <property name="font">
            <font>
             <family>Arial</family>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_addAndEdit">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>333</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Data</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QGridLayout" name="gridLayout_4" columnstretch="10,5">
              <item row="3" column="0" colspan="2">
               <widget class="QLineEdit" name="le_p2_addDepartment">
                <property name="font">
                 <font>
                  <family>Arial</family>
                 </font>
                </property>
               </widget>
              </item>
              <item row="4" column="0" colspan="2">
               <widget class="QLabel" name="lb_p2_costUnit">
                <property name="font">
                 <font>
                  <family>Arial</family>
                 </font>
                </property>
                <property name="text">
                 <string>CostUnit</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="lb_p2_name">
                <property name="font">
                 <font>
                  <family>Arial</family>
                 </font>
                </property>
                <property name="text">
                 <string>Name*</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="le_p2_addName">
                <property name="font">
                 <font>
                  <family>Arial</family>
                 </font>
                </property>
               </widget>
              </item>
              <item row="5" column="0" colspan="2">
               <widget class="QComboBox" name="cb_p2_addCostUnit">
                <property name="font">
                 <font>
                  <family>Arial</family>
                 </font>
                </property>
               </widget>
              </item>
              <item row="6" column="0" colspan="2">
               <widget class="QLabel" name="lb_p2_companyLocation">
                <property name="text">
                 <string>Company Location*</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0" colspan="2">
               <widget class="QLabel" name="lb_p2_department">
                <property name="font">
                 <font>
                  <family>Arial</family>
                 </font>
                </property>
                <property name="text">
                 <string>Department</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="le_p2_addPhone">
                <property name="font">
                 <font>
                  <family>Arial</family>
                 </font>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="lb_p2_phone">
                <property name="font">
                 <font>
                  <family>Arial</family>
                 </font>
                </property>
                <property name="text">
                 <string>Phone*</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QComboBox" name="cb_p2_addCompanyLocation"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Acions</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="1" column="0">
               <widget class="QPushButton" name="pb_p2_clearData">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Clear all data from the form below. This action cannot be undone.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Clear</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="pb_p2_addNew">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Saves changes to the contact&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Save</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="pb_p2_back">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Go back to the caller overview&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Back</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="pb_p2_delete">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Delete this Caller&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>333</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../ams.qrc"/>
 </resources>
 <connections/>
</ui>
